spec/acceptance/validate_puppet_spec.rb

```diff
@@ -47,7 +47,7 @@ class foo {
         its(:exit_status) { is_expected.to eq(0) }
         its(:stderr) { is_expected.to match(syntax_spinner_text) }
         its(:stderr) { is_expected.to match(lint_spinner_text) }
-        its(:stdout) { is_expected.to have_no_output }
+        its(:stdout) { is_expected.to match(%r{Target does not contain any files to validate}) }
```

Comment:
It seems very strange that adding a completely new class, which doesn't affect `.pp` files, and no test fixture changes, has caused this test to be modified

Reply:
The full message says it does not contain any EPP files to validate.

Reply:
Fair enough.